src/eval/reduction.rs

```diff
@@ -706,6 +708,7 @@ fn apply_continuation<F: LurkField>(
     let cont_witness = &mut witness.conts;
 
     let control = match cont.tag {
+        ContTag::Cproc => unreachable!(),
```

Comment:
is this necessary? If so, can we equip the `unreachable!` call with a message justifying that logic?